docs/en/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -180,6 +179,8 @@ inputs:
           ssl_assert_fingerprint: "22:F7:FB:84:1D:43:3E:E7:BB:F9:72:F3:D8:97:AD:7C:86:E3:08:42" #optional
 ----
 
+WARNING: All versions up to 1.14.0 (included) only allow one output per type. So if the `output.type` chosen by a user is `elasticsearch`, then the user can only configure one output for it.
```

Comment:
Adding this warning. See related issue: https://github.com/elastic/elastic-serverless-forwarder/issues/721.